manuscript/ms.tex

```diff
@@ -268,7 +268,7 @@ \subsection{PDB mining results}
 \subsection{Small-scale kinase expression test in \emph{E. coli}}
 
 A panel containing the 96 kinase domain constructs selected through our semi-automated method, was tested for expression in \emph{E. coli}. 
-From this initial test, 68 kinase domains showed detectable expression (yield of more than 2 ng/$\mu$l eluate) (Table~\ref{expression_table}).
+From this initial test, 52 kinase domains showed reasonable expression (yield of more than 2 mg/L in the eluate) (Table~\ref{expression_table}).
```

Comment:
You have "2 mg/L in the eluate".  Do you mean "2 mg/L culture"?  

It might be good to revert this change, or to include both numbers:
"yield of more than 2 ng/$\mu$L eluate, which corresponds to 2 mg/L culture".